### PR TITLE
Merged event-based Hubspot updates into batch update (company snapshot)

### PR DESCRIPTION
**Pull request by alexey-rise**
## Description
Merged event-based Hubspot updates into batch update (company snapshot)

## Motivation and Context
We need to get rid of event-based Hubspot updates using "fake" system contact in order to fix https://github.com/Rise-Vision/rise-vision-apps/issues/1332
See https://trello.com/c/KZmsduDo for more details.

## How Has This Been Tested?
Tested using BQ console. New fields have already been added to `rise-core-log:customerData.company`

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
N/A

### CODE REVIEW

**Comment by tejohnso** [comment]
@alexey-rise will you also be changing the hubspot-connector companyChanges.sql and associated job?

**Comment by alexey-rise** [comment]
@tejohnso yes: https://github.com/Rise-Vision/hubspot-connector/pull/88 ,  https://github.com/Rise-Vision/core/pull/571

File: src/main/resources/sql/companySnapshot.sql [hunk 1]
```diff
@@ -99,15 +108,15 @@ activeDisplays as
 (
 select display_id as displayId
 from `client-side-events.Uptime_Events.events`
-where showing = true and connected = true and ts >= timestamp(DATE_SUB(CURRENT_DATE(), INTERVAL 7 DAY))
+where showing = true and connected = true and ts >= timestamp(DATE_SUB(CURRENT_DATE(), INTERVAL 29 DAY))
```
Comment (tejohnso) [inline src/main/resources/sql/companySnapshot.sql]: Why does this one have to change to 29 days ago?

Comment (alexey-rise) [reply src/main/resources/sql/companySnapshot.sql]: @tejohnso please see 
https://rise-vision-inc.slack.com/archives/CCELMSZFZ/p1572466114298600
https://rise-vision-inc.slack.com/archives/CCELMSZFZ/p1572466350299500

Comment (tejohnso) [reply src/main/resources/sql/companySnapshot.sql]: Thanks

**Pull request merged**